### ISSUE TITLE
Move NSTextStorageDelegate to an extension

### ISSUE THREAD

**Issue by neoneye** [kuyawa/MarkdownEditor#2]
IMHO Instead of

    class ViewController: NSViewController, NSTextStorageDelegate {

Then do

    class ViewController: NSViewController {
        // ...
    }
    extension ViewController: NSTextStorageDelegate {
        func textStorage()
    }

**Comment by kuyawa** [comment]
Good point but since the extension is just one little function in this particular case I'm leaving everything in one place. If the project ever grows then I'll start to refactor everything moving all pieces to their individual classes as I usually do.

Thanks for the tip.
